src/content/lodash/function/bindKey/lodash.js

```diff
@@ -0,0 +1,20 @@
+// https://lodash.com/docs/#bindKey
+import _, { bindKey } from 'lodash'
```

Comment:
Does it make sense to omit the `_` import (and/or mention directly in a comment in the code that placeholder option isn't implemented in the replacement?)

```suggestion
import { bindKey } from 'lodash'
```

Reply:
Definitely, this is a leftover of when I was using other lodash methods in the method, will push the change and merge, thanks

Reply:
I'm a bit surprised it wasn't caught by lint though: I moved to `next lint` as part of the transition but I might have to refine the rules